--- raine-0.50.11/makefile.false	2008-03-14 23:24:04.000000000 +0100
+++ raine-0.50.11/makefile	2008-10-08 00:34:36.000000000 +0200
@@ -20,7 +20,7 @@
 # RAINE_DEBUG = 1
 
 # Be verbose ?
-# VERBOSE = 1
+VERBOSE = 1
 
 # Use asm video core ? (comment to use C core)
 ASM_VIDEO_CORE = 1
@@ -204,9 +204,9 @@
 SDL = 1
 
    prefix = $(DESTDIR)/usr
-   bindir = $(prefix)/games
-   sharedir = $(prefix)/share/games
-   mandir = $(prefix)/man/man6
+   bindir = $(prefix)/bin
+   sharedir = $(prefix)/share
+   mandir = $(sharedir)/man/man6
    rainedata = $(sharedir)/raine
 ifndef SDL
    langdir = $(rainedata)/languages
@@ -258,7 +258,6 @@
 
 ifndef SDL
 ifeq ("$(shell if [ -e /usr/include/vga.h ] || [ -e /usr/local/include/vga.h ]; then echo yes; fi)","yes")
-GFX_SVGALIB=1
 endif
 endif
 
@@ -305,7 +304,7 @@
 OBJDIR = $(OSTYPE)-sdl
 endif
 
-include cpuinfo
+#include cpuinfo
 
 ifdef X86_64
 OBJDIR := $(OBJDIR)64
@@ -388,25 +387,18 @@
 
 ifdef RAINE32
 # when starting a game -> black screen if -O > 1 (bug in uint64 calculation)
-CFLAGS = -O1
+#CFLAGS = -O1
 else
 # Seems to work now, at least with the sdl version ? (to be tested with windows !)
-CFLAGS = -O3
+CFLAGS = $(RPMFLAGS)
 endif
 
 CFLAGS += $(INCDIR) \
 	$(DEFINE) \
-	$(_MARCH) \
-	-Wno-trigraphs \
-	-fschedule-insns2 \
-	-funroll-all-loops \
-	-fexpensive-optimizations \
-	-ffast-math \
-	-w \
-	-fomit-frame-pointer
+	$(_MARCH)
 
 # This is required for gcc-2.9x (bug in -fomit-frame-pointer)
-CFLAGS_MCU = $(_MARCH) -O3 -fexpensive-optimizations # switches for the 68705 mcus
+CFLAGS_MCU = $(CFLAGS)s # switches for the 68705 mcus
 
 ifdef RAINE_UNIX
 CFLAGS += -pipe
@@ -1156,7 +1148,7 @@
 	./makedep $(OBJDIR) $(OBJS) > make.dep
 
 cpuinfo:
-	@sh ./detect-cpu
+#	@sh ./detect-cpu
 
 include make.dep
 
@@ -1207,8 +1199,6 @@
 	sh -c "if [ -f hiscore.dat ]; then $(INSTALL_DATA) hiscore.dat $(rainedata); fi"
 	sh -c "if [ -f history.dat ]; then $(INSTALL_DATA) history.dat $(rainedata); fi"
 	$(INSTALL_DATA) config/cheats.cfg $(rainedata)
-	$(INSTALL_DATA) raine.desktop $(prefix)/share/applications
-	$(INSTALL_DATA) raine.png $(prefix)/share/pixmaps
 
 install_dirs:
 	$(MD) -pv $(bindir) $(rainedata) $(langdir) $(romdir) $(artdir) $(emudxdir) $(prefix)/share/pixmaps $(prefix)/share/applications $(bitmaps_dir) $(fonts_dir)
